free-programming-books-tr.md

```diff
@@ -26,6 +26,7 @@
 ### Android
 
 * [Android Geleceği Yazanlar](https://gelecegiyazanlar.turkcell.com.tr/konu/android)
+* [Android Türkçe PDF](http://umiitkose.com/wp-content/uploads/2015/08/AndroidStudio.pdf)
```

Comment:
Could you please add PDF in brackets at the end of the line instead?
This is a convention in the repository.

This appears to be HTTPS accessible, could the link also be replaced with an HTTPS one.

```suggestion
* [Android Türkçe](https://umiitkose.com/wp-content/uploads/2015/08/AndroidStudio.pdf) (PDF)
```